src/components/alert/Alert.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {ReactElement, ReactNode} from "react";
+import {IconAlertCircle, IconCircleCheck, IconCircleX, IconInfoCircle, IconX} from "@tabler/icons-react";
+import "./Alert.style.scss"
+
+export interface AlertType {
+
+    children?: ReactNode | ReactNode[]
+    title: ReactNode
+    //defaults to info
+    variant?: "primary" | "secondary" | "success" | "warning" | "error"
+    //defaults to true
+    icon?: boolean
+    //defaults to false
+    dismissible?: boolean
+    onClose?: (event: React.MouseEvent<HTMLSpanElement, MouseEvent>) => void
+}
+
+const IconVariants = {
+    "primary": <IconInfoCircle/>,
+    "secondary": <IconInfoCircle/>,
+    "success": <IconCircleCheck/>,
+    "warning": <IconAlertCircle/>,
+    "error": <IconCircleX/>
+
+}
+
+const Alert: React.FC<AlertType> = (props) => {
+
+    const {variant = "primary", dismissible = false, icon = true, title, onClose = (event) => {}, children} = props
+
+    return <div className={`alert alert--${variant}`}>
+        <div className={"alert__header"}>
+            <div className={"alert__header-wrapper"}>
+                {icon ? <AlertIcon variant={variant}/> : null}
+                <span className={"alert__heading"}>{title}</span>
+            </div>
+            {dismissible ? <span className={"alert__dismissible"} onClick={event => onClose(event)}><IconX/></span> : null}
```

Comment:
Just an idea, but would this work?
```suggestion
            {dismissible ? <span className={"alert__dismissible"} onClick={onClose}><IconX/></span> : null}
```

Reply:
Hmm could be. I will try and come back to you

Reply:
> Just an idea, but would this work?

Yep, appreciate that 👍🏻